pkgs/applications/terminal-emulators/tilix/default.nix

```diff
@@ -16,17 +16,18 @@
 , glib
 , wrapGAppsHook
 , libunwind
+, appstream
 }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "tilix";
-  version = "unstable-2019-10-02";
+  version = "1.9.4";
 
   src = fetchFromGitHub {
     owner = "gnunn1";
     repo = "tilix";
-    rev = "ffcd31e3c0e1a560ce89468152d8726065e8fb1f";
-    sha256 = "1bzv7xiqhyblz1rw8ln4zpspmml49vnshn1zsv9di5q7kfgpqrgq";
+    rev = "${version}";
```

Comment:
```suggestion
    rev = version;
```

Reply:
I am happy to close https://github.com/NixOS/nixpkgs/pull/115133
I am not sure who is working on what.
I just need one or two (preferably all 3) working D compilers, (dub and dub2nix for the time being) in relation to the things that burns my D related time on Nix*.
D needs as wide and active a Nix* community as possible. Let me give a shout of thanks to @i077 who has been an active contributor in the earlier related pull requests, and make sure that he is apprised of what is happening here. @i077 again thank you very much ... looks as though there is and will be more to be done in future ... look forward to hearing you